config/api_defaults.yml

```diff
@@ -26,6 +26,6 @@ semver:
   objc: '0.6.0'
   java: '0.6.0'
   nodejs: '0.6.0'
-  python: '1.0.18'
+  python: '0.7.0'
```

Comment:
This commit seems to be squashed into a previous commit changing the same file. In that commit, the version is "0.7.6" instead of "0.7.0". Are we going to use 0.7.0?